### PR TITLE
Implement Chartbeat for MAPs and Live Radio Pages

### LINKED ISSUE #5521: Implement Chartbeat for MAP and Live Radio Pages
**Is your feature request related to a problem? Please describe.**
Implement Chartbeat for MAP and Live Radio pages.

**Describe the solution you'd like**
MAPs -  https://jira.dev.bbc.co.uk/browse/WSMEDIA-92
Live Radio - https://jira.dev.bbc.co.uk/browse/WSMEDIA-147

The current [implementation](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ChartbeatAnalytics/utils/index.js#L31) only supports front-pages and articles in the `getType` function. We can update this utility to get support for MAPs and live radio pages.

To get Chartbeat working for MAP pages, we need to update the current `ChartbeatAnalytics` component to accept MAP and Live radio as valid page types and use it in a new `MapChartbeat` component which will implement the `cbq` property.

| Property | MAP implementation | Live Radio Implementation |
|--------|-----------------------|-----------------------------|
| `uid` | Unchanged | Unchanged |
| `domain` | Unchanged | Unchanged |
| `useCanonical` | Unchanged | Unchanged |
| `sections` | Add MAP to `buildSections` in utils | Add Live Radio to `buildSections` in utils |
| `bbc_hid` | Unchanged | Unchanged |
| `title` | Add MAP to `getTitle` | Add LiveRadio to `getTitle` |
| `type` | Add MAP `getType` in utils | Add Live radio to `getType` in utils |

***Details on `sections`***
MAPs
We want to capture the service e.g "Afrique" and then the section name which we get from Ares e.g. "afrique" so this value would be "Afrique - afrique"
 
Then repeat the same entries again appended with "- MAP"
 
Then capture the category if there is one 
Service - <category name> - category e.g 
"Afrique - feature-category"
11:54
This would be
"< service> , <service> - <section name>, < Service> -MAP , <service> - <section name> -MAP, <service> - <category name>-category"
e.g. "Afrique, Afrique - afrique, Afrique - MAP, Afrique - afrique - MAP, Afrique - feature-category"

And in the case of live radio - because we don't have section or category for radio this would be something like:
"Arabic, Arabic - , Arabic - unknown, Arabic - unknown, Arabic - unknown";

So live radio will always be:
“[Service], [Service] - , [Service] - unknown, [Service] - unknown, [Service] - unknown”

**Testing notes**

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by pharingee**
Resolves #5521

**Overall change:** 
Implement Chartbeat for MAPs and Live Radio Pages

**Code changes:**
- Update Chartbeat utils to accept `MAP` and `media` as valid page types
- Add Chartbeat component to `RadioMain` and `MediaAssetPage`

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/ChartbeatAnalytics/utils/index.js [hunk 1]
```diff
@@ -35,24 +36,53 @@ export const getType = (pageType, shorthand = false) => {
...
+        buildSectionItem(serviceCap, appendCategory(categoryName)),
+      ].join(', ');
+    case 'media':
+      return [
+        serviceCap,
+        buildSectionItem(serviceCap, ''),
+        buildSectionItem(serviceCap, 'unknown'),
+        buildSectionItem(serviceCap, 'unknown'),
+        buildSectionItem(serviceCap, 'unknown'),
+      ].join(', ');
```
Comment (karinathomasbbc) [inline src/app/containers/ChartbeatAnalytics/utils/index.js:77]: Is this just temporary until we have values for Live Radio?

**Review by karinathomasbbc — APPROVED** [review]

File: src/app/containers/ChartbeatAnalytics/utils/index.js (hunk 1 above)
Comment (pharingee) [reply src/app/containers/ChartbeatAnalytics/utils/index.js:77]: No, this is the expected value for Live Radio.

**Review by karinathomasbbc — APPROVED** [review]
LGTM 👍

**Review by rhenshaw56 — CHANGES_REQUESTED** [review]
@pharingee storybook is broken for the RadioPage when the [chartbeat toggle](https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/toggles/index.js#L4) is enable locally.

You would have to wrap the MediaAssetPage story component with the `<UserContextProvider />` as it's been [done for articles](https://github.com/bbc/simorgh/blob/latest/src/app/containers/ArticleMain/index.stories.jsx#L23-L25).

Screen Shot 2020-02-24 at 17 53 10

**Review by ryanmccombe — APPROVED** [review]
👍

**Review by rhenshaw56 — APPROVED** [review]

**Pull request merged**